frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/RelationshipEdge/RelationshipEdge.tsx

```diff
@@ -41,6 +31,24 @@ export const RelationshipEdge: FC<Props> = ({
       <BaseEdge
         id={id}
         path={edgePath}
+        markerStart={
+          data?.cardinality
+            ? data?.isHighlighted
+              ? 'url(#zeroOrOneRightHighlight)'
+              : 'url(#zeroOrOneRight)'
```

Comment:
className is not applied to markers because these are defined with outer component, I think.
That's why I define markers for non-highlighted or highlighted, but it is redundant 🥲